.changeset/six-chicken-brush.md

```diff
@@ -1,26 +0,0 @@
----
-"@justeattakeaway/pie-assistive-text": patch
-"@justeattakeaway/pie-checkbox-group": patch
-"@justeattakeaway/generator-pie-component": patch
-"@justeattakeaway/pie-cookie-banner": patch
-"@justeattakeaway/pie-notification": patch
-"@justeattakeaway/pie-webc-testing": patch
-"@justeattakeaway/pie-icon-button": patch
-"@justeattakeaway/pie-form-label": patch
-"@justeattakeaway/pie-text-input": patch
-"@justeattakeaway/pie-checkbox": patch
-"@justeattakeaway/pie-textarea": patch
-"@justeattakeaway/pie-divider": patch
-"@justeattakeaway/pie-spinner": patch
-"@justeattakeaway/pie-button": patch
-"@justeattakeaway/pie-switch": patch
-"@justeattakeaway/pie-modal": patch
-"@justeattakeaway/pie-toast": patch
-"@justeattakeaway/pie-card": patch
-"@justeattakeaway/pie-chip": patch
-"@justeattakeaway/pie-link": patch
-"@justeattakeaway/pie-tag": patch
----
-
-[Added] - pie-css as a dev dependency
-[Changed] - updated component generator
```

Comment:
@fernandofranca this is the only file with two changes I believe. I wonder if that is contributing to the missing dash 🤔